README.md

```diff
@@ -34,35 +150,40 @@ $ ansible-playbook -i localhost, --connection=local \
 [![asciicast](https://asciinema.org/a/161278.png)](https://asciinema.org/a/161278)
 
 
-## Deploy new cluster + KubeVirt
+## Deploy new Kubernetes or OpenShift cluster and KubeVirt with Lago
 
+This section describes how to deploy KubeVirt or OpenShift cluster with KubeVirt installed on it using Lago.
 
-### Requirements
-Install depending roles, and export `ANSIBLE_ROLES_PATH`
-```
-$ ansible-galaxy install -p $HOME/galaxy-roles -r requirements.yml
-$ export ANSIBLE_ROLES_PATH=$HOME/galaxy-roles
-```
-For OpenShift deployment clone [**OpenShift Ansible project**](https://github.com/openshift/openshift-ansible)
-```
-$ git clone https://github.com/openshift/openshift-ansible
-```
+### Parameters
 
-### Kubernetes
-Preparing Kubernetes cluster and deploy KubeVirt on it.
-- Add your master and nodes to `inventory` file.
-- Run ansible playbook `# ansible-playbook -i inventory deploy-kubernetes.yml`.
+| Name             |  Value        | Description                            |
+| ---------------- | ------------- | -------------------------------------- |
+| cluster\_type    | `kubernetes`, `openshift` | Desired cluster            |
+| mode             | `release`, `dev` | If `dev` it will build KubeVirt from sources |
```

Comment:
Don't we agree to drop dev mode?

Reply:
@gbenhaim not yet, we were just discussing it at #62

Reply:
+1